google-cloud-logging/lib/google/cloud/logging/async_writer.rb

```diff
@@ -62,11 +63,13 @@ class AsyncWriter
 
         ##
         # @private Item in the log entries queue.
-        QueueItem = Struct.new(:entries, :log_name, :resource, :labels) do
+        QueueItem = Struct.new(:entries, :log_name, :resource, :labels,
+                               :partial_success) do
```

Comment:
I don't think this is correct. Each item in the queue should not manage this value, it should be set on the writer object and used for every write request.

Reply:
Oops, I misunderstood the previous comment. Just changed `partial_success` to be an instance variable.